src/peft/helpers.py

```diff
@@ -146,3 +154,102 @@ def check_if_peft_model(model_name_or_path: str) -> bool:
         is_peft_model = False
 
     return is_peft_model
+
+
+def get_module(name, opt_model):
```

Comment:
All this code here can be removed, right?

Reply:
Yes, removed, thanks for pointing that out. I wonder if I screwed up a rebase somewhere. Anyway, gone now.